src/components/structures/GroupView.js

```diff
@@ -246,8 +252,36 @@ export default React.createClass({
         });
     },
 
+    _onAvatarSelected: function(e) {
+    },
+
+    _onAvatarChange: function(newAvatarUrl) {
+        const newProfileForm = Object.assign(this.state.profileForm, { avatar_url: newAvatarUrl });
+        this.setState({
+            profileForm: newProfileForm,
+        });
+    },
+
     _onSaveClick: function() {
-        // TODO: There's no API to edit group profile info yet.
+        this.setState({saving: true});
+        MatrixClientPeg.get().setGroupProfile(this.props.groupId, this.state.profileForm).then((result) => {
+            this.setState({
+                saving: false,
+                editing: false,
+                summary: false,
+            });
+            return this._loadGroupFromServer(this.props.groupId);
+        }).catch((e) => {
+            this.setState({
+                saving: false,
+            });
+            const ErrorDialog = sdk.getComponent("dialogs.ErrorDialog");
+            console.error("Failed to save group profile", e);
+            Modal.createDialog(ErrorDialog, {
+                title: _t('Error'),
+                description: _t('Failed to update group'),
+            });
+        }).done();
```

Comment:
No, but I figured I would keep adding them for a little bit just in case it all goes hrribly wrong and we have to back out bluebird, but *shrug*